python/lsst/cp/pipe/cpDefects.py

```diff
@@ -84,6 +88,11 @@ class MeasureDefectsTaskConfig(pipeBase.PipelineTaskConfig,
         allowed={'STDEV': "Use a multiple of the image standard deviation to determine detection threshold.",
                  'VALUE': "Use pixel value to determine detection threshold."},
     )
+    fitAmpGradient = pexConfig.Field(
+        dtype=bool,
+        doc=("Fit out the radial gradient per amplifier."),
```

Comment:
-> "Fit out the focal plane radial gradient ...".  Also doesn't need parentheses.